core/src/main/scala/org/apache/spark/ui/jobs/AllStagesPage.scala

```diff
@@ -30,7 +30,8 @@ import org.apache.spark.ui.{UIUtils, WebUIPage}
 private[ui] class AllStagesPage(parent: StagesTab) extends WebUIPage("") {
   private val sc = parent.sc
   private val subPath = "stages"
-  private def isFairScheduler = parent.isFairScheduler
+  // Show pool information for only live UI
+  private def isFairScheduler = sc.isDefined && parent.isFairScheduler
```

Comment:
It would be better to fix `StageTab.isFairScheduler` and remove this method.

Reply:
Done